src/chrome/content/rules/Kalooga.com.xml

```diff
@@ -1,18 +1,65 @@
 <!--
-	(www.): works; mismatched, CN: *.squarespace.com
+	cert mismatch:
+		- cdn.kalooga.com
+		- cdnnet.kalooga.com
+		- publishing.cdnnet.kalooga.com
+		- cdnnet0.kalooga.com
+		- cdnnet1.kalooga.com
+		- cdnnet2.kalooga.com
+		- cdnnet3.kalooga.com
+		- cloudfront.kalooga.com
+		- exceptions.kalooga.com
 
+	chain issues:
+		- company.kalooga.com
+		- demo.kalooga.com
+		- m.kalooga.com
+		- top.kalooga.com
 -->
 <ruleset name="Kalooga.com (partial)">
 
+	<target host="kalooga.com" />
+	<target host="www.kalooga.com" />
+
+	<target host="api.kalooga.com" />
+	<target host="apps.kalooga.com" />
+	<target host="assets.kalooga.com" />
+	<target host="backoffice.kalooga.com" />
+	<target host="bowser.kalooga.com" />
+	<target host="control.kalooga.com" />
+	<target host="corp.kalooga.com" />
+	<target host="dev1.kalooga.com" />
+	<target host="events.kalooga.com" />
+	<target host="geo.kalooga.com" />
+	<target host="grab.kalooga.com" />
+	<target host="img.kalooga.com" />
+	<target host="img0.kalooga.com" />
+	<target host="img1.kalooga.com" />
+	<target host="img2.kalooga.com" />
+	<target host="img3.kalooga.com" />
+	<target host="img4.kalooga.com" />
+	<target host="nagios.kalooga.com" />
+	<target host="origin.kalooga.com" />
+	<target host="placeholder.kalooga.com" />
 	<target host="publishing.kalooga.com" />
+	<target host="reporter.kalooga.com" />
+	<target host="signup.kalooga.com" />
+	<target host="static.kalooga.com" />
+
 	<target host="geo.kaloo.ga" />
 	<target host="img.kaloo.ga" />
 	<target host="publishing.kaloo.ga" />
 
+	<test url="http://kalooga.com/" />
+	<test url="http://www.kalooga.com/" />
+
 	<test url="http://publishing.kaloo.ga/wid/16513.js?loc=http%3A%2F%2Fwww.bz-berlin.de%2Fberlin%2Fber-zweite-anzeige-gegen-jochen-grossmann" />
 	<test url="http://img.kaloo.ga/thumb?url=http%3A%2F%2Fwww.shz.de%2Fimg%2Fwirtschaft%2Fcrop11177506%2F8806138589-cv16_9-h335%2Furn-newsml-dpa-com-20090101-151111-99-05409-large-4-3.jpg&amp;md5val=911f772734b61479440330f1140d13b4&amp;key=00f8a403a132c924d4f37b5df7818f1454f668f8&amp;size=192x140" />
 	<test url="http://publishing.kaloo.ga/acct/4455.js" />
 	<test url="http://geo.kaloo.ga/json/" />
+	
+	<rule from="^http://kalooga\.com/"
```

Comment:
Please keep a comment for the issue on this host.

Reply:
cert mismatch is noted now